x-pack/plugin/ql/src/main/java/org/elasticsearch/xpack/ql/util/SpatialCoordinateTypes.java

```diff
@@ -56,10 +52,6 @@ public long pointAsLong(double x, double y) {
             return (yi & 0xFFFFFFFFL) | xi << 32;
         }
 
-        public SpatialPoint pointAsPoint(Point point) {
-            return makePoint(point.getX(), point.getY());
```

Comment:
Now only one remaining call to makePoint. But that one is almost certainly needed by the doc-values version, for which it was originally written.